runtime/src/test/java/org/corfudb/runtime/clients/ManagementHandlerTest.java

```diff
@@ -79,6 +81,13 @@ private Layout getBasicLayout(List<String> nodes) {
         return new Layout(nodes, nodes, Collections.singletonList(segment), 0L, DEFAULT_UUID);
     }
 
+    private EpochCheck getEnumIgnoreEpoch(boolean ignoreEpoch) {
+        if (ignoreEpoch) {
+            return EpochCheck.IGNORE;
+        }
+        return EpochCheck.CHECK;
+    }
```

Comment:
Instead of introducing a new method, you can just use `handlingSuccessful ? EpochCheck.IGNORE ? EpochCheck.CHECK`

Reply:
Done, thanks!